template-manager-tag/src/test/java/com/sequenceiq/cloudbreak/DefaultCostTaggingServiceTest.java

```diff
@@ -100,22 +101,62 @@ public void testMergeTagsShouldReturnWithAnUnion() {
         Assert.assertEquals("apple4", "apple4");
     }
 
+    @Test
+    public void testAccountTagUserTagEqualityNoError() {
+        Map<String, String> envMap = new HashMap<>();
+        envMap.put("apple1", "apple1");
+        envMap.put("apple2", "apple2");
+        envMap.put("owner", "owner");
+        Map<String, String> requestTag = new HashMap<>();
+        requestTag.put("pear1", "pear1");
+        requestTag.put("owner", "owner");
+
+        CDPTagGenerationRequest tagRequest = tagRequest("AWS", new HashMap<>(), envMap, requestTag);
+        underTest.prepareDefaultTags(tagRequest);
+    }
+
+    @Test
+    public void testAccountTagUserTagConflictGeneartesError() {
```

Comment:
testAccountTagUserTagConflictGene**rat**esError

Reply:
Fixed in the latest push.